parsovani znamek z "title" bunky namisto "<span>"

kryglik opened the pull request:
Na nasi skole se pouziva i procentni hodnoceni, ktere se objevuje v bunce. Muzou tam tak byt hodnoty "1 2 1 90". Nicmene v "title" dane bunky je to jiz spravne prevedene na znamky: "1 2 1 1".

kryglik committed:
parsovani znamek z "title" bunky namisto "<span>"

Na nasi skole se pouziva i procentni hodnoceni, ktere se objevuje v bunce. Muzou tam tak byt hodnoty "1 2 1 90". Nicmene v "title" dane bunky je to jiz spravne prevedene na znamky: "1 2 1 1".

sarahyzova reviewed (changes requested):
Samotný výpočet průměru vypadá v pořádku, nicméně tato změna způsobuje, že simulované známky nejsou zahrnuty do výpočtu průměru. Poprosím vás ještě o opravení této chyby.

kryglik committed:
oprava simulovanych znamek

simulovane znamky prochazim standardne pres querySelectorAll a vyuzivam vlastnosti hledani konkretni tridy (querySelectorAll("span.simulace"))

kryglik commented:
Opraveno, tuhle cast kodu jsem bohuzel opomenul, diky za postrehnuti.

Commit:
Bugfix - simulace známek v prázdném políčku

sarahyzova reviewed (approved):
Ještě tam byl menší bug, ale ten jsem opravil sám.
Děkuji za příspěvek, změny se již brzy projeví v produkční verzi.
